build.sbt

```diff
@@ -46,6 +48,8 @@ lazy val atomLibraries = (project in file("."))
     setReleaseVersion,
     commitReleaseVersion,
     tagRelease,
+    // For non cross-build projects, use releaseStepCommand("publishSigned")
+    releaseStepCommandAndRemaining("+publishSigned"),
     releaseStepCommand("publishSigned"),
```

Comment:
do we need both lines? I think just the +publishSigned is required?

Reply:
Good catch! I'll get rid of it...